validate.js

```diff
@@ -8,13 +8,24 @@ const {
   THEMES_SCHEME
 } = require('./definitions.js');
 
+const RECORDS_CONFORMANCE_CLASS = "http://wis.wmo.int/spec/wcmp/2/conf/core"; // todo

```

Comment:
This should be the OGC API - Records - Part 1: Core conformance class, I think, but the example doesn't have it.
See also: https://github.com/ESA-EarthCODE/open-science-catalog-metadata/issues/233#issuecomment-2450514379